docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Set up Automatic Agent Updates
+description: Describes how to set up automatic agent updates.
+---
+
+Teleport supports automatic agent updates for systemd-based Linux distributions
+using `apt`, `yum`, and `zypper` package managers, as well as Kubernetes
+clusters. 
+
+This guide explains how to enable automatic updates for Teleport agents on
+cloud-hosted Teleport Enterprise clusters.
+
+## How it works
+
+When automatic updates are enabled, a Teleport updater is installed alongside
+each Teleport agent. The updater communicates with the Teleport Proxy Service to
+determine when an update is available. When an update is available, the updater
+will update the Teleport agent during the next maintenance window. However, if a
+critical update is availble, the Teleport agent will be updated outside the
+regular maintenance window.
+
+## Prerequisites
+
+- A cloud-hosted Teleport Enterprise cluster. If you do not have one, [sign
+  up](https://goteleport.com/signup) for a free trial or consult the [Update
+  Reference](./reference.mdx) to read about manual updates or automatic agent
+  updates for self-hosted clusters.
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components in your cluster.
+
+- (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+Systemd agents enrolled into automatic updates can only install versions
+present in their package repositories. As Teleport 14 won't be published to
+`stable/v13`, those agents will require manual intervention to be upgraded to
+the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).
+
+Alternatively, you can use the `stable/rolling` channel, which contains
```

Comment:
I'm not really sure what the use case for `stable/rolling` was supposed to be, but for Teleport Cloud we instruct users to use the `stable/cloud` channel.

Reply:
Thanks! The two paragraphs about release channels for older Teleport versions probably shouldn't be in this page. I've removed them, and also added a subsection to the reference about choosing a release channel, based on the text in `docs/pages/includes/repo-channels.mdx`.